GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -1025,7 +1025,7 @@ private bool DrawVisibleGraph(int start, int end)
 
                 // Get the x,y value of the current item's upper left in the cache
                 int curCacheRow = (_cacheHeadRow + rowIndex - _cacheHead) % _cacheCountMax;
-                int x = 0;
+                const int x = 0;
                 int y = curCacheRow * _rowHeight;
 
                 var laneRect = new Rectangle(0, y, Width, _rowHeight);
```

Comment:
(If changing, use x, y)

Reply:
`y` can't be a constant because it's sub-expressions are not constant.

Reply:
but the following:
var laneRect = new Rectangle(x, y,

Reply:
`const` must be a compile time constant, currently a literal value such as `1.234` or `"foo"`.